modoboa/core/password_hashers/advanced.py

```diff
@@ -49,6 +52,9 @@ class MD5CRYPTHasher(PasswordHasher):
 
     This scheme can't be considered as secure anymore.
     """
+    name = "md5crypt"
+    label = "md5crypt"
```

Comment:
Two properties in the base class would be better. The scheme name can be guessed from the class name and we could add a weak flag too.